tests/bwc/test_upgrade.py

```diff
@@ -44,8 +44,22 @@
         VersionDef('5.7.x', []),
         VersionDef('5.8.x', []),
         VersionDef('5.9.x', []),
-        VersionDef('latest-nightly', [])
+        VersionDef('5.10.x', []),
     ),
+    (
+        VersionDef('5.0.x', []),
+        VersionDef('5.1.x', []),
+        VersionDef('5.2.x', []),
+        VersionDef('5.3.x', []),
+        VersionDef('5.4.x', []),
+        VersionDef('5.5.x', []),
+        VersionDef('5.6.x', []),
+        VersionDef('5.7.x', []),
+        VersionDef('5.8.x', []),
+        VersionDef('5.9.x', []),
+        VersionDef('5.10.x', []),
+        VersionDef('latest-nightly', [])
```

Comment:
Wouldn't 5.10 -> nightly fail? I thought any upgrade to the nightly is problematic at the moment

Reply:
The problem is only related to rolling upgrades. A full cluster restart upgrade works fine from any 5.x to 6.x (the blocker here is index compatibility, not any CrateDB version handshake -> no mixed version cluster).